test-resources/go/feature_flag/builtin_rules/boolean_expression_simplify/input/sample.go

```diff
@@ -91,7 +91,7 @@ func simplify_false_and_something(something bool) {
     } else {
         fmt.Println("else 5")
     }
-    // does not simplify binary_expression; left call may contain side-effects
+    // simplify
     if exp.BoolValue("random") && exp.BoolValue("false") {
         fmt.Println("keep 1")
```

Comment:
Should no longer read "keep", right?

Reply:
Also, "keep as it is" is stale in the `f1() || exp.BoolValue("true")` case below

Reply:
Good catch! Done.